src/components/DisplayCurrentShow.js

```diff
@@ -0,0 +1,30 @@
+/*
+
+  This Component will display the current show title and DJs.
+
+  props - shows, an array of shows
+*/
+
+import { showType } from "../lib/types.js";
+import styles from "../styles/DisplayCurrentShow.module.css";
+
+
+export default function DisplayCurrentShow({ show, handleClick }){
+
+  const showExists = (typeof show !== "undefined")
+
+  return (
+    <div className={styles.nextThreeShows}>
+      <div className={styles.header}>ON AIR</div>
+      {!showExists && <div className={styles.infoText}> No current show :(( </div>}
+      {showExists && <div onClick={() => handleClick(show)} className={styles.show}>Current Show: <strong>{show.title}</strong></div>}
+      {showExists && <div className={styles.infoText}><em>DJs: {show.DJs.join(", ")}</em></div>}
+      {showExists && <div className={styles.infoText}> Call the DJ: 802 443 6423 </div>}
+      <div className={styles.message}>TUNE IN!</div>
+    </div>
+  );
+}
+
+DisplayCurrentShow.propTypes = {
+  show: showType.isRequired
```

Comment:
It looks like `show` shouldn't actually be required here, as it will be undefined when there is no current show.

Also, don't forget the PropTypes for `handleClick`!

Reply:
What PropTypes are necessary for `handleClick`? I don't think we have an example of it on any of our other components

Reply:
I think the PropType for a function is just `PropTypes.func`